packages/modules/cra-template-desktop-viewer/template/.env

```diff
@@ -1,5 +1,5 @@
 # ---- Configuration ----
-ITWIN_VIEWER_SCOPE ="imodelaccess:read offline_access imodels:read projects:read realitydata:read"
+ITWIN_VIEWER_SCOPE ="imodelaccess:read imodels:read projects:read realitydata:read"
```

Comment:
Removed the "offline_access" scope from the environment as when trying to describe how to add it, we realized that this was a ghost scope that is added automagically to native client, but is never listed in the dev portal. The scope is automatically added to the list of required scopes by the client https://github.com/iTwin/auth-clients/blob/main/packages/electron/src/main/Client.ts#L93 so it is now completely hidden to reduce confusion.

Reply:
great find!